### ISSUE TITLE
URL read error by ffmpeg when streaming WebM file

### ISSUE THREAD

**Issue by ZayneHuang** [vbence/stream-m#50]
Hi, thank you for your nice work in streaming WebM format files.

However, I met some problem somehow when running the process according to the guideline.

The program package is from the latest release: [https://github.com/vbence/stream-m/releases/tag/v0.2.2](https://github.com/vbence/stream-m/releases/tag/v0.2.2).

The streaming server is open with `java -jar stream-m.jar server.properties.sample` without any change.

When I run the following command to run the ffmpeg as suggested by _TESTING THE INSTALLATION_ of README
```
ffmpeg -re -i ./file_example_WEBM_1920_3_7MB.webm -vcodec copy -acodec copy -f webm http://localhost:8080/publish/first?password=secret
```

**The error message was as following, it shows that there was some problems with the URL. But I cannot find any mistake or wrong in the command... Would you mind providing some suggestion on how to solve it? Thanks a lot!**
```
av_interleaved_write_frame(): Unknown errortime=00:00:00.00 bitrate=N/A speed=N/A
Error writing trailer of http://localhost:8080/publish/first?password=secret: Error number -10053 occurred
frame=   14 fps=0.0 q=-1.0 Lsize=      50kB time=00:00:00.43 bitrate= 941.4kbits/s speed=1.05x
video:82kB audio:0kB subtitle:0kB other streams:0kB global headers:4kB muxing overhead: unknown
[http @ 000002077444fa00] URL read error: Error number -10053 occurred
Conversion failed!
```

Here is the full message by ffmpeg:
```
ffmpeg version 4.4.1-essentials_build-www.gyan.dev Copyright (c) 2000-2021 the FFmpeg developers
  built with gcc 11.2.0 (Rev1, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-lzma --enable-zlib --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-sdl2 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-libass --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libgme --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libtheora --enable-libvo-amrwbenc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-librubberband
  libavutil      56. 70.100 / 56. 70.100
  libavcodec     58.134.100 / 58.134.100
  libavformat    58. 76.100 / 58. 76.100
  libavdevice    58. 13.100 / 58. 13.100
  libavfilter     7.110.100 /  7.110.100
  libswscale      5.  9.100 /  5.  9.100
  libswresample   3.  9.100 /  3.  9.100
  libpostproc    55.  9.100 / 55.  9.100
Input #0, matroska,webm, from './file_example_WEBM_1920_3_7MB.webm':
  Metadata:
    ENCODER         : Lavf57.83.100
  Duration: 00:00:30.54, start: 0.000000, bitrate: 960 kb/s
  Stream #0:0: Video: vp8, yuv420p(progressive), 1920x1080, SAR 1:1 DAR 16:9, 30 fps, 30 tbr, 1k tbn, 1k tbc (default)
    Metadata:
      HANDLER_NAME    : L-SMASH Video Handler
      COMPATIBLE_BRANDS: mp42mp41isomavc1
      MAJOR_BRAND     : mp42
      MINOR_VERSION   : 0
      ENCODER         : Lavc57.107.100 libvpx
      DURATION        : 00:00:30.036000000
  Stream #0:1: Audio: vorbis, 48000 Hz, stereo, fltp (default)
    Metadata:
      HANDLER_NAME    : L-SMASH Audio Handler
      COMPATIBLE_BRANDS: mp42mp41isomavc1
      MAJOR_BRAND     : mp42
      MINOR_VERSION   : 0
      ENCODER         : Lavc57.107.100 libvorbis
      DURATION        : 00:00:30.543000000
Output #0, webm, to 'http://localhost:8080/publish/first?password=secret':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Video: vp8, yuv420p(progressive), 1920x1080 [SAR 1:1 DAR 16:9], q=2-31, 30 fps, 30 tbr, 1k tbn, 1k tbc (default)
    Metadata:
      HANDLER_NAME    : L-SMASH Video Handler
      COMPATIBLE_BRANDS: mp42mp41isomavc1
      MAJOR_BRAND     : mp42
      MINOR_VERSION   : 0
      ENCODER         : Lavc57.107.100 libvpx
      DURATION        : 00:00:30.036000000
  Stream #0:1: Audio: vorbis, 48000 Hz, stereo, fltp (default)
    Metadata:
      HANDLER_NAME    : L-SMASH Audio Handler
      COMPATIBLE_BRANDS: mp42mp41isomavc1
      MAJOR_BRAND     : mp42
      MINOR_VERSION   : 0
      ENCODER         : Lavc57.107.100 libvorbis
      DURATION        : 00:00:30.543000000
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (copy)
Press [q] to stop, [?] for help
av_interleaved_write_frame(): Unknown errortime=00:00:00.00 bitrate=N/A speed=N/A
Error writing trailer of http://localhost:8080/publish/first?password=secret: Error number -10053 occurred
frame=   14 fps=0.0 q=-1.0 Lsize=      50kB time=00:00:00.43 bitrate= 941.4kbits/s speed=1.05x
video:82kB audio:0kB subtitle:0kB other streams:0kB global headers:4kB muxing overhead: unknown
[http @ 000002077444fa00] URL read error: Error number -10053 occurred
Conversion failed!
```

**Comment by mbattista** [comment]
Hi.

can you try again with the current release: https://github.com/vbence/stream-m/releases/tag/untagged-7c8ac906f86d0ad0fa5c
If it still persists I can look into it.

Thanks.

**Comment by ZayneHuang** [comment]
Thank you for your rapid and kind reply.

I have tried again the current release you mentioned on my Windows PC. The problem still exiests. But I tried it on another Windows PC with the same version of ffmpeg and stream-m. It ran successfully.

So I guess that it might because of something about the system settings or proxy settings...

**I removed the system environment path: HTTP_proxy and HTTPS_proxy. The problem was fixed then.**

Anyway, thanks for your help again. I really appreciate the stream-m project. It really helps me a lot~
